### ISSUE TITLE
[Bug]: Misleading warning title for duplicate file name

### ISSUE THREAD

**Issue by chan-j-d** [commons-app/apps-android-commons#5181]
### Summary

When uploading a file with caption that causes a duplicate file name, the title of the warning says `Duplicate Image Found` which seems to imply that the image itself is duplicated instead of the file name.

### Steps to reproduce

1. Upload an image with the caption field `asd`

### Expected behaviour

`Duplicate File Name found` as title

### Actual behaviour

`Duplicate Image Found`

### Device name

Pixel XL

### Android version

API 30

### Commons app version

master

### Screen-shots

![image](https://user-images.githubusercontent.com/65345505/225334650-c9922dc8-42d4-4fc2-9804-1e03c4e9eee3.png)

### Would you like to work on the issue?

None

**Comment by RohitLakra299** [comment]
Hey, I would like to work in this. 
Can I get assigned.

**Comment by nicolas-raoul** [comment]
@RohitLakra299 Sorry I did not see this in time!

@shankarpriyank You should have asked to be assigned before starting working on it.

As it is a very small modification I guess we can go with Shankar for now, sorry Rohit and hopefully you can find another issue you want to work on! :-)

**Comment by shankarpriyank** [comment]
Sorry I apologize @nicolas-raoul and @RohitLakra299 would keep this is mind next time.
